app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ProductActivity.java

```diff
@@ -71,6 +80,51 @@ protected void onCreate(Bundle savedInstanceState) {
         if (!Utils.isHardwareCameraInstalled(this)) {
             mButtonScan.setVisibility(View.GONE);
         }
+
+        BottomNavigationView bottomNavigationView = (BottomNavigationView)findViewById(R.id.bottom_navigation);
+        BottomNavigationViewHelper.disableShiftMode(bottomNavigationView);
+        bottomNavigationView.setOnNavigationItemSelectedListener(item -> {
+
+            switch (item.getItemId()) {
+                case R.id.bookmark:
+                    // implementation of bookmark wii be here
+                    Toast.makeText(ProductActivity.this,"Bookmark",Toast.LENGTH_SHORT).show();
+                    break;
+                case R.id.share:
+                    String shareUrl = " " + getString(R.string.website_product) + mState.getProduct().getCode();
+                    Intent sharingIntent = new Intent();
+                    sharingIntent.setAction(Intent.ACTION_SEND);
+                    sharingIntent.setType("text/plain");
+                    String shareBody = getResources().getString(R.string.msg_share) + shareUrl;
+                    String shareSub = "\n\n";
+                    sharingIntent.putExtra(Intent.EXTRA_SUBJECT, shareSub);
+                    sharingIntent.putExtra(Intent.EXTRA_TEXT, shareBody);
+                    startActivity(Intent.createChooser(sharingIntent, "Share using"));
+                    break;
+                case R.id.translation:
```

Comment:
This code can stay in, however, for now, lets comment it out because there is no point in it being there